shpy

```diff
@@ -7,22 +7,33 @@ createSpy() {
     fi
 
     # shellcheck disable=SC2039
-    local val output error_output
+    local val output error_output stub_dir outputs_dir last_arg
+    for last_arg; do true; done
 
+    _shpyInit
+
+    stub_dir=$(realpath "$_shpy_spies_dir")
+    outputs_dir=${stub_dir}/outputs/${last_arg}
+    mkdir -p "${outputs_dir}"
```

Comment:
This codebase uses `shpy_ensure_dir` to create the appropriate directory, and doesn't call `realpath`. You should be able to do `shpy_ensure_dir "$_shpy_spies_dir/outputs/$spy_name"` in place of these three lines

Reply:
Fixed